packages/salesforcedx-vscode-core/src/orgBrowser/metadataOutlineProvider.ts

```diff
@@ -138,13 +138,13 @@ export class MetadataOutlineProvider
 
 export function parseErrors(error: string | any): Error {
   try {
+
     const errMsg = typeof error === 'string' ? error : error.message;
-    const sanitized = errMsg.substring(
-      errMsg.indexOf('{'),
-      errMsg.lastIndexOf('}') + 1
-    );
+    const sanitized = extractJsonObject(errMsg);
```

Comment:
It looks like the JSON string is being sanitized and parsed by all the files. Could we include the parsing functionality in the new utility method as well?